docs/docs/javadoc.md

```diff
@@ -0,0 +1,10 @@
+# Javadoc
+
+Choose a module to view its generated Javadoc:
+
+- [streams-bootstrap-cli](../javadoc/streams-bootstrap-cli/index.html){target=_blank}
```

Comment:
Is it possible to build a combined JavaDoc instead? https://stackoverflow.com/questions/63287292/how-to-combine-multiple-javadoc-into-one-using-gradle